nautobot/docs/user-guide/core-data-model/cloud/cloudaccount.md

```diff
@@ -0,0 +1,3 @@
+# Cloud Accounts
+
+Cloud accounts provide a way to track account details for public and private cloud [providers](../dcim/manufacturer.md) (AWS, Azure, GCP, DigitalOcean and OpenStack etc) in Nautobot. Each cloud account has to have a unique name and an account number and you may optionally assign a secrets group to it.
```

Comment:
Probably would be good to update `manufacturer.md` to explicitly indicate that it's recommended to use for cloud providers. Maybe also update `circuits/provider.md` to state that for cloud providers, `manufacturer` is recommended instead/in addition.